libs/api/domains/temporary-voter-registry/src/lib/temporaryVoterRegistry.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@nestjs/common'
+import { logger } from '@island.is/logging'
+import { ApolloError } from 'apollo-server-express'
+import {
+  TemporaryVoterRegistryApi,
+  VoterRegistryControllerFindOneRequest,
+} from '../../gen/fetch/apis'
+
+const handleError = async (error: any) => {
+  logger.error(JSON.stringify(error))
```

Comment:
Also, would be better to import logger with dependency injection, right? maybe a service somewhere to handle the errors?

Reply:
Yes, moving this into the class and using injection seems cleaner to me.
Can you elaborate on how you envision a service to handle the errors?

Reply:
I think this might overlap our previous discussion regarding:
`libs/api/domains/party-letter-registry/src/lib/partyLetterRegistry.service.ts`
Will assume we handle this in a separate PR and resolve.
Please correct me if I'm misunderstanding

Reply:
Nope, sounds good to me 👍 

Regarding a service, I was just thinking something like:

```
@Injectable()
export class NestErrorService {
	handleError() {
   		...
	}
}


this.nestErrorService.handleError(...)
```

Not sure how much we want this function to be customizable but it's just my take on it